CedarJava/src/test/resources/duplicate_policies.cedar

```diff
@@ -0,0 +1,13 @@
+@id("Policy")
```

Comment:
I don't see this test case used above, but IIRC that this _will not_ be an error. The `@id()` annotation is given special meaning in the CLI, but not the Cedar library itself. So this annotation isn't actually setting the policy id here. The two policies are assigned default distinct ids, like "policy0" and "policy1".

Reply:
Ah I see. I had initially used this to test that the Rust parsing would output a set with a single member since the policy content was identical, but I see that that's not the case, and multiple policies with different IDs but the same source are returned. I removed the tests, but forgot to remove this file as well.